resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,20 @@
+<!-- resources/views/layouts/app.blade.php -->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Laravel Quickstart</title>
+        <link rel="stylesheet" href="{{ mix('/css/app.css') }}">
```

Comment:
1. Nghiên cứu https://laracasts.com/discuss/channels/general-discussion/what-is-the-l5-equivalent-of-htmlstylecssdefaultcss?page=1 để extend css và js e nhé.
2. Nghiên cứu Laravel mix để quản lý css và js em nhé https://laravel.com/docs/5.5/mix